modules/cluster/main.tf

```diff
@@ -31,6 +33,10 @@ resource "ibm_satellite_cluster" "create_cluster" {
     }
   }
 
+  lifecycle {
+    ignore_changes = [resource_group_id]
+  }
+
```

Comment:
This seems a bit separate from the stated change. Just want to make sure it's intentional on this PR.